identity-mapping.json

```diff
@@ -69,14 +69,59 @@
 		"locationHierarchyForProfiling": {
 			"value": "zone,postalCode"
 		},
-		"addressLine1": {
-			"value": "addressLine1"
+		"presentAddressLine1": {
```

Comment:
normally in identity mapping json file, we can change only the value, the keys as it is will be used in the code base

Reply:
Yes sir corrected.